Makefile

```
@@ -190,6 +190,18 @@ clint: build/.ran-cmake
 clint-full: build/.ran-cmake
 	+$(BUILD_CMD) -C build clint-full
 
+clang-tidy: generate-compilation-db
+	clang-tidy --config-file .clang-tidy -p build $(FILE)
+
+clang-tidy-full: generate-compilation-db
+	bash -C build clang-tidy-full
```

should this be `$(BUILD_CMD)` ?
```suggestion
	+$(BUILD_CMD) -C build clang-tidy-full
```